bot/models/configs.py

```diff
@@ -0,0 +1,20 @@
+from .model import Model
+
+
+class Config(Model):
+    id: int
+    guild_id: int
+    min_xp: int
+    max_xp: int
+    xp_boost: int
+    custom_role_log_channel_id: int | None
+
+    @classmethod
+    async def ensure_exists(cls, guild_id: int):
+        query = """
+            INSERT INTO configs (guild_id)
+                 VALUES ($1)
+            ON CONFLICT (guild_id)
+          DO UPDATE SET guild_id = configs.guild_id
+              RETURNING *"""
+        return await cls.fetchrow(query, guild_id)
```

Comment:
We shouldn't need this function here, we can insert a row into the table when the bot is added to a guild and perhaps run a check when the bot starts that a config exists for each guild the bot is in